controller/lib/hal/hal.h

```diff
@@ -99,14 +99,12 @@ enum class PinMode : uint8_t {
 enum class VoltageLevel : uint8_t { HAL_CONSTANT(HIGH), HAL_CONSTANT(LOW) };
 
 enum class AnalogPin {
-  // PATIENT_PRESSURE is an MPXV5004 sensor, reading an absolute pressure
-  // value at the patient.
+  // MPXV5004DP pressure sensors.
   //
-  // {INFLOW,OUTFLOW}_PRESSURE_DIFF are MPXV5004 sensors, reading a pressure
-  // differential across a venturi.  They let us measure volumetric flow into
-  // and out of the patient.
+  // PATIENT_PRESSURE reads an absolute pressure value at the patient.
   //
-  // Details at https://bit.ly/3bFwhpP
+  // {INFLOW,OUTFLOW}_PRESSURE_DIFF, read a differential across a venturi.
+  // They let us measure volumetric flow into and out of the patient.
```

Comment:
No need for that bit.ly link anymore?

Reply:
No, it's out of date. https://respiraworks.slack.com/archives/C0100MWE4V8/p1587936746167900

Edwin gave me the Correct Link, however, which will make an appearance in a later PR.